docs/articles/instances.md

```diff
@@ -0,0 +1,38 @@
+
+See [Instance Life Cycle](instance-lifecycle.md) for more information on instance life cycle and goals.
+
+### Instances in the API
+The `v2/apps` and `v2/tasks` API endpoints do not provide any means of direct interaction with instances. For historical reasons, these endpoints list apps or tasks respectively, and only allow interacts with them (not instances). The `v2/pods` endpoint on the other hand was build with instances as an abstraction for Mesos tasks, and thus only allows direct interaction with instances, not with tasks.
+
+### Instance Ids
+The relation between instances and their Tasks is reflected in their names. An *app* with name `/my-service` will result in instances named `my-service.<UUID>`. The first Task associated with this instance will be named `my-service.instance-a41beca1-fa49-11e8-93c4-269daf75ef3f._app.1` where `_app` designates the container as an anonymous container. The `.1` reveals the Task is the first task incarnation of this instance. Accordingly, `my-service.instance-<UUID>._app.42` will tell you that this is task incarnation number 42. This means 41 previous tasks for this instance became terminal, either because they exited normally or abnormally, or were killed by Marathon during a scale or upgrade operation.
+
+A *pod* instance will have named containers that inherit the names from the respective PodDefinition, e.g. `my-service.instance-395c02eb-25c5-4a3b-8a42-3d2635264058.rails.1` and `my-service.instance-395c02eb-25c5-4a3b-8a42-3d2635264058.sidecar.1`.
+
+## How can I use Instances?
+Instances are created based on [Service](services.md) specifications which can be either an [AppDefinition](applications.md) or a [PodDefinition](pods.md). Instances are implicitly created according to their App or Pod definition, which contains an `instances` property that designates the required amount of instances.
+
+## Which related concepts should I understand?
+When upgrading a Service during a [Deployment](deployments.md) from one version to another, Marathon can create more instances than specified in either Service definition. It will do so if allowed via the [upgradeStrategy](upgrade-strategy.md) configuration.
+When an instance is running on an agent which has disconnected from the cluster, it will eventually be reported as unreachable. According to the Service’s [unreachableStrategy](unreachable-strategy.md), Marathon might schedule a replacement, even though the unreachable instance might still be running. When an unreachable instance becomes reachable again, Marathon will scale down accordingly to the required amount of instances. It will decide which instance shall be killed based on the [killSelection](kill-selection.md).
+
+## Links
```

Comment:
As noted by @zen-dog earlier, both Links and Examples point to resources which don't exist at the moment. We should either drop those links, or add the resources.

Reply:
yeah... I think we should create the docs (links that are missing) with an empty to be documented note... but I as assume that we will discuss this in our 1 on 1.